katas/content/single_qubit_gates/index.md

```diff
@@ -365,6 +365,65 @@ All the basic gates we will be covering in this kata are part of the <a href="ht
     "solutionPath": "./sign_flip_on_zero/solution.md"
 })
 
+@[section]({
+    "id": "identity_gate",
+    "title": "Identity Gate"
+})
+
+The identity gate is mostly here for completeness, at least for now. It will come in handy when dealing with multi-qubit systems and multi-qubit gates. It is represented by the identity matrix, and does not affect the state of the qubit.
+  
+<table>
+<tr>
+<th>Gate</th>
+<th>Matrix</th>
+<th>Ket-Bra</th>
+<th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
+<th>Q# Documentation</th>
+</tr>
+<tr>
+<td>$I$</td>
+<td>$\begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}$</td>
+<td>$|0\rangle\langle0| + |1\rangle\langle1|$</td>
+<td>$I|\psi\rangle = |\psi\rangle$</td>
+<td><a href=\"https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.i\">I</a></td>
+</tr>
+</table>
+
+@[section]({
+    "id": "hadamard_gate",
+    "title": "Hadamard Gate"
+})
+
+The **Hadamard** gate is an extremely important quantum gate. Unlike the previous gates, applying the Hadamard gate to a qubit in a computational basis state puts that qubit into a superposition.
+Like the Pauli gates, the Hadamard gate is self-adjoint.
+
+<table>
+<tr>
+<th>Gate</th>
+<th>Matrix</th>
+<th>Ket-Bra</th>
+<th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
+<th>Applying to basis states</th>
+<th>Q# Documentation</th>
+</tr>
+<tr>
+<td>$H$</td>
+<td>$\begin{bmatrix} \frac{1}{\sqrt{2}} & \frac{1}{\sqrt{2}} \\ \frac{1}{\sqrt{2}} & -\frac{1}{\sqrt{2}} \end{bmatrix} = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 & 1 \\ 1 & -1 \end{bmatrix}$</td>
+<td>$|0\rangle\langle+| + |1\rangle\langle-|$</td>
+<td>$H|\psi\rangle = \alpha|+\rangle + \beta|-\rangle = \frac{\alpha + \beta}{\sqrt{2}}|0\rangle + \frac{\alpha - \beta}{\sqrt{2}}|1\rangle$</td>
+<td>$H|0\rangle = |+\rangle \\
```

Comment:
\\ doesn't seem to work in the playground environment to make multi-line formulas, other tables in this kata put each basis state effect in a separate formula and added `<br>` for newlines